flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPOperatorTest.java

```diff
@@ -73,7 +73,8 @@ public Integer getKey(Event value) throws Exception {
 					false,
 					keySelector,
 					IntSerializer.INSTANCE,
-					new NFAFactory()),
+					new NFAFactory(),
+					true),
```

Comment:
Ah, I see now what the parameter does. Even if you don't migrate you still set it to true. The parameter basically indicates "would have been a keyed operator whether we actually have legacy state or not". That's why it's always false when restoring from the old non-keyed operator.

We just use it in the `restoreState()` method so when we don't have legacy state the value is never actually checked.